com.microsoft.java.test.runner/src/main/java/com/microsoft/java/test/runner/common/MessageUtils.java

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
+* Copyright (c) 2018 Microsoft Corporation and others.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Eclipse Public License v1.0
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v10.html
+*
+* Contributors:
+*     Microsoft Corporation - initial API and implementation
+*******************************************************************************/
+
+package com.microsoft.java.test.runner.common;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class MessageUtils {
+
+    public static TestMessageItem createWithName(String title, String nameValue) {
```

Comment:
See this book for how to construct object in design: https://www.safaribooksonline.com/library/view/effective-java-3rd/9780134686097/ch2.xhtml